db/fixtures/miq_product_features.yml

```diff
@@ -6449,7 +6449,7 @@
     :identifier: storage_service_control
     :children:
     - :name: Refresh
-      :description: Refresh relationships and power states for all items related to Physical Storages
+      :description: Refresh the storage manager of this storage service
```

Comment:
We agreed to leave these right?

Reply:
it's from the previous old commit, I haven't squashed them yet

Reply:
removed the `miq_product_features.yml` changes